generators/server/templates/src/main/resources/config/_application-dev.yml

```diff
@@ -187,6 +187,17 @@ jhipster:
                 # Token is valid 24 hours
                 tokenValidityInSeconds: 86400
     <%_ } _%>
+    <%_ if (authenticationType === 'uaa') { _%>
+    security:
+        client-authorization:
+            # change this depending on your authorization server
+            <%_ if (applicationType !== 'uaa') { _%>
+            accessTokenUri: http://<%= uaaBaseName.toLowerCase() %>:9999/oauth/token
```

Comment:
We could do the same move here as with uaaBasename like uaaPort. I will take a look